elements/pfe-sass/maps/_typography.scss

```diff
@@ -40,16 +50,22 @@ $pf-type-sizing: (
    FontSize--xs:   pf-font-prem($pf-global--FontSize--xs),  // 12px
 );
 
-/// General component usage such as accordion headers and CTAs. Use with calc()
-/// @example scss - variable example
-///   var(--pf-c-FontSize)
-///
-$pf-type-sizing--component: (
-  FontSize:                 #{pfe-var(FontSize--md )},  // 16px
-);
+/// Sets of typographical block-level HTML elements
+/// @see {mixin} pfe-content 
+
+$pf-content-set: (h1, h2, h3, h4, h5, h6, p, small);
```

Comment:
Thinking out loud but `small` is a modifier and best practices for it suggest it only alter font-size and nothing else.  Does it really belong in this set?

Reply:
It's a good question, I was following suite with was PF core does: 
https://github.com/patternfly/patternfly/blob/master/src/patternfly/components/Content/content.scss#L57
but we could leave it out for now